signer/src/transaction_coordinator.rs

```diff
@@ -1360,6 +1350,110 @@ where
 
         Ok(wallet)
     }
+
+    /// Assesses the total fees paid for any outstanding sweep transactions in
+    /// the mempool which may need to be RBF'd. If there are no sweep
+    /// transactions which are spending the signer's UTXO, then this function
+    /// will return [`None`].
+    ///
+    /// TODO: This method currently blindly assumes that the mempool transactions
+    /// are correct. Maybe we need some validation?
+    #[tracing::instrument(skip_all, fields(%chain_tip, signer_utxo = %signer_utxo.outpoint))]
```

Comment:
We probably want to remove the `chain_tip` from here.

Reply:
Actually wasn't needed at all, must have forgotten to remove it at some point